CHANGES.rst

```diff
@@ -303,6 +310,8 @@ New Features
 
 - ``astropy.modeling``
 
+- ``astropy.cosmology``
```

Comment:
@adrn I guess it's the result of a rebase gone bad, and I hadn't checked the diff before pushing. Thanks for catching.